README.md

```diff
@@ -1,4 +1,5 @@
-# Synopsys Action
+# DEPRECATED: Synopsys Action
+**NOTE:** This action has been deprecated and will not work after February 14, 2025. It is recommended that you migrate to our new <a href="https://github.com/marketplace/actions/black-duck-security-scan">Black Duck Security Scan</a>.Instructions can be found at <a href="https://documentation.blackduck.com/bundle/bridge/page/documentation/c_using-github-action.html">BlackDuckDoc's</a>.
```

Comment:
Should be -
This action has been deprecated and will not work after February 14, 2025. It is recommended that you migrate to our new <a href="https://github.com/marketplace/actions/black-duck-security-scan">Black Duck Security Scan</a>. Instructions can be found at <a href="https://documentation.blackduck.com/bundle/bridge/page/documentation/c_using-github-action.html">Black Duck Docs</a>.

Reply:
I see the same text in read me file